app/views/batch/_metadata.html.erb

```diff
@@ -0,0 +1,50 @@
+<h2>Individual Titles  </h2>
+<div id="" class="well">
+  <div class="row">
+    <div class="col-sm-8">
+      <h3>Applies to individual files uploaded</h3>
+      <%= content_tag :p, t('sufia.batch.help.title'), class: "help-block" %>
+      <% @batch.generic_files { |a,b| a.label.downcase <=> b.label.downcase }.each_with_index  do |gen_f, index| %>
```

Comment:
https://github.com/projecthydra/sufia/blob/v6.0.0/app/views/batch/_metadata.html.erb#L7

The version of sufia we are using uses #sort!. Time to upgrade sufia?